<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>niftk::IGIUltrasoundOverlayWidget</class>
 <widget class="QWidget" name="niftk::IGIUltrasoundOverlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="m_LeftImageCheckBox">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>left image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QmitkDataStorageComboBox" name="m_LeftImageCombo" native="true"/>
     </item>
     <item>
      <widget class="QmitkDataStorageComboBox" name="m_RightImageCombo" native="true"/>
     </item>
     <item>
      <widget class="QCheckBox" name="m_RightImageCheckBox">
       <property name="text">
        <string>right image</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_TrackingLabel">
       <property name="text">
        <string>tracking</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QmitkDataStorageComboBox" name="m_TrackingCombo" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_OpacityLabel">
       <property name="text">
        <string>overlay opacity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_OpacitySlider">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_3DViewCheckBox">
       <property name="text">
        <string>3D view</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="m_Splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="m_OverlayViewerLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="niftk::SingleUltrasoundWidget" name="m_LeftOverlayViewer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="niftk::SingleUltrasoundWidget" name="m_RightOverlayViewer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <zorder>m_LeftOverlayViewer</zorder>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QmitkRenderWindow" name="m_3DViewer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkRenderWindow</class>
   <extends>QWidget</extends>
   <header location="global">QmitkRenderWindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>niftk::SingleUltrasoundWidget</class>
   <extends>QWidget</extends>
   <header>niftkSingleUltrasoundWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QWidget</extends>
   <header>QmitkDataStorageComboBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
